src/test/java/io/openapitoools/jackson/dataformat/hal/ser/HALBeanSerializerPolymorphicWithPropertyIT.java

```diff
@@ -1,51 +1,49 @@
 package io.openapitoools.jackson.dataformat.hal.ser;
 
-import static org.junit.Assert.assertEquals;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 
 import com.fasterxml.jackson.annotation.JsonSubTypes;
 import com.fasterxml.jackson.annotation.JsonTypeInfo;
 import com.fasterxml.jackson.annotation.JsonTypeInfo.As;
 import com.fasterxml.jackson.annotation.JsonTypeInfo.Id;
 import com.fasterxml.jackson.annotation.JsonTypeName;
 import com.fasterxml.jackson.databind.ObjectMapper;
+
 import io.openapitools.jackson.dataformat.hal.HALLink;
 import io.openapitools.jackson.dataformat.hal.HALMapper;
 import io.openapitools.jackson.dataformat.hal.annotation.EmbeddedResource;
 import io.openapitools.jackson.dataformat.hal.annotation.Link;
 import io.openapitools.jackson.dataformat.hal.annotation.Resource;
-import java.net.URI;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import org.junit.Test;
 
 public class HALBeanSerializerPolymorphicWithPropertyIT {
 
   ObjectMapper om = new HALMapper();
 
   @Test
   public void testSerializationForResourceWithEmbeddableList() throws Exception {
-    @Resource
-    class TopResource {
-
-      public String id = "1";
-
-      @Link public HALLink self = new HALLink.Builder(URI.create("/top/1")).build();
-
-      @Link("child")
-      public List<HALLink> childResourcesLink =
-          Arrays.asList(
-              new HALLink.Builder(URI.create("/top/1/child/1")).build(),
-              new HALLink.Builder(URI.create("/top/1/child/2")).build());
-
-      @EmbeddedResource("child")
-      public Collection<ChildResource> children =
-          Arrays.asList(new ChildResource("1"), new OtherChildResource("2", "Max"));
-    }
+      @Resource
+      @SuppressWarnings("unused")
+      class TopResource {
```

Comment:
Something about the indentation here, right? Seems it was 2 spaces but now is 4 except inside of the `TopResource` its still 2.. (seems like the project is missing a checkstyle run or something :-) )

Reply:
I've fixed the indentation in this file. I've set the width to 4 spaces now, which is what most other files seemed to be using (and it's what's commonly used for Java files). I didn't find any other files that had this issue though I didn't check all of them. 

I've also added an [`.editorconfig`](https://editorconfig.org/) file to this project. This allows per-project code style configuration and most IDEs can recognize it (usually through a plugin). It's currently quite minimal, with only the 4-space indentation width configured. But it can be updated as needed and should provide consistent styling configuration for anyone using this project.